views/partials/graphmenu.hbs

```diff
@@ -1,15 +1,11 @@
-<section class="ui raised segment">
     <div class="ui inverted blue vertical pointing menu">
-        <a id="burndown"   class= "item" href="/graphs">Burndown</a>
+        <a id="burndown"   class= "item" href="/burndown">Burndown</a>
```

Comment:
might be worth putting a stub here, and also a stub route for sprintReport also, at the moment the graph component of burndown and sprintReport are identical, but if we get the accompanying table up and running that accompany either report, they would differ then